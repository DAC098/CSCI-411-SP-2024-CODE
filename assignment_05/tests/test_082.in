906 575
5
30
56
56
61
75
78
106
132
148
175
176
177
182
215
237
240
242
242
249
250
252
257
258
290
294
306
309
311
336
377
391
394
402
406
412
424
433
444
456
470
477
485
499
509
518
519
525
537
539
542
542
544
575
593
599
600
600
611
616
616
642
653
659
665
665
680
688
701
716
721
730
739
775
775
776
808
823
836
838
841
864
881
899
911
918
923
923
927
930
931
931
943
979
979
1027
1043
1076
1084
1106
1108
1110
1117
1139
1165
1179
1204
1211
1228
1254
1261
1282
1286
1299
1322
1331
1335
1342
1350
1375
1382
1410
1412
1416
1428
1449
1457
1489
1502
1518
1534
1543
1557
1558
1563
1588
1589
1621
1627
1632
1649
1665
1668
1671
1674
1688
1699
1721
1722
1746
1756
1810
1812
1816
1830
1831
1831
1873
1891
1898
1909
1916
1927
1929
1940
1943
1953
2015
2025
2030
2034
2034
2046
2063
2067
2073
2088
2092
2107
2113
2116
2132
2169
2184
2194
2197
2246
2254
2267
2276
2282
2286
2297
2305
2331
2336
2337
2342
2348
2377
2391
2401
2406
2423
2439
2443
2466
2476
2490
2491
2500
2505
2518
2524
2566
2606
2614
2618
2641
2649
2665
2676
2745
2754
2764
2767
2799
2810
2812
2825
2828
2835
2838
2855
2868
2868
2870
2873
2897
2913
2920
2958
2959
2984
3032
3034
3038
3045
3057
3065
3069
3112
3129
3139
3147
3154
3170
3213
3225
3230
3252
3264
3299
3300
3301
3316
3321
3321
3354
3359
3363
3388
3412
3443
3460
3484
3494
3494
3497
3513
3514
3515
3568
3570
3621
3630
3647
3656
3675
3681
3709
3715
3746
3766
3792
3800
3801
3802
3802
3806
3815
3816
3857
3862
3862
3874
3894
3923
3930
3935
3944
3951
3956
3963
3974
3995
3999
4016
4024
4028
4033
4034
4049
4061
4063
4076
4084
4089
4091
4099
4108
4113
4120
4121
4140
4154
4155
4220
4232
4254
4277
4302
4313
4322
4343
4350
4388
4389
4390
4396
4402
4416
4417
4435
4448
4449
4461
4474
4482
4501
4506
4515
4516
4522
4526
4539
4550
4554
4590
4595
4600
4605
4619
4628
4674
4678
4691
4692
4704
4705
4715
4719
4721
4728
4728
4761
4762
4765
4767
4767
4807
4809
4814
4830
4863
4870
4898
4907
4912
4934
4954
4961
4961
4967
4969
4972
4995
5025
5033
5043
5045
5090
5103
5107
5107
5120
5120
5165
5169
5181
5197
5213
5227
5228
5287
5289
5307
5313
5327
5329
5340
5362
5368
5369
5372
5402
5417
5429
5437
5458
5462
5464
5474
5480
5489
5530
5549
5554
5587
5588
5591
5606
5627
5631
5648
5653
5655
5718
5730
5739
5746
5753
5772
5776
5781
5797
5817
5822
5830
5833
5834
5861
5866
5875
5878
5883
5890
5918
5926
5938
5941
5954
5974
6003
6020
6038
6058
6069
6081
6082
6091
6095
6098
6105
6110
6122
6160
6177
6191
6192
6200
6222
6248
6251
6256
6263
6265
6269
6275
6277
6284
6301
6306
6323
6333
6379
6381
6387
6411
6415
6422
6427
6438
6447
6449
6453
6460
6461
6464
6470
6477
6482
6510
6525
6526
6541
6553
6564
6579
6580
6584
6588
6589
6591
6599
6622
6634
6642
6647
6675
6682
6707
6715
6718
6726
6732
6750
6768
6776
6779
6792
6802
6821
6843
6849
6853
6856
6863
6867
6868
6870
6873
6886
6902
6911
6913
6915
6920
6936
6946
6956
6973
6981
6984
6993
7003
7007
7021
7026
7029
7031
7032
7035
7041
7051
7053
7055
7074
7075
7077
7077
7098
7108
7116
7122
7123
7123
7138
7141
7158
7165
7175
7188
7202
7219
7219
7226
7232
7238
7238
7269
7275
7286
7298
7302
7303
7304
7313
7316
7318
7327
7336
7338
7347
7348
7348
7354
7355
7358
7361
7364
7366
7384
7385
7403
7405
7409
7410
7434
7438
7439
7443
7445
7445
7467
7499
7504
7523
7527
7554
7555
7560
7564
7581
7609
7623
7624
7649
7650
7661
7662
7678
7688
7691
7695
7695
7702
7710
7712
7712
7713
7725
7730
7747
7768
7777
7794
7796
7813
7816
7830
7843
7845
7861
7866
7868
7884
7884
7895
7918
7918
7931
7956
7960
7966
7975
7984
7993
8000
8005
8019
8027
8028
8051
8058
8083
8121
8135
8141
8151
8168
8171
8211
8214
8215
8217
8235
8251
8282
8296
8318
8384
8414
8431
8451
8462
8476
8488
8489
8498
8546
8549
8595
8599
8605
8610
8611
8612
8613
8617
8629
8642
8644
8646
8653
8664
8664
8669
8681
8682
8687
8697
8698
8701
8713
8715
8716
8717
8721
8732
8743
8764
8776
8799
8836
8841
8853
8861
8862
8883
8886
8908
8913
8913
8920
8932
8965
8966
8976
8977
8979
8986
8992
8995
8996
9019
9039
9043
9047
9050
9052
9084
9090
9113
9130
9135
9136
9142
9163
9191
9198
9205
9230
9234
9246
9251
9293
9294
9300
9310
9320
9336
9346
9350
9363
9393
9415
9419
9422
9439
9440
9444
9448
9454
9458
9459
9484
9486
9489
9523
9524
9537
9552
9573
9573
9585
9589
9594
9608
9613
9614
9619
9628
9642
9650
9659
9673
9680
9680
9684
9689
9700
9704
9723
9729
9741
9743
9750
9754
9760
9766
9770
9784
9789
9806
9821
9830
9835
9865
9867
9873
9888
9890
9890
9893
9893
9912
9915
9921
9923
9926
9943
9943
9947
9961
9962
9965
9966
9973
9974
9977
9978
9980
9990
9991
9997
