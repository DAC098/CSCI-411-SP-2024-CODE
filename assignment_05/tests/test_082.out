Enter the number of hotels and the ideal number of miles to travel per day: Enter 906 hotel distances each on a separate line: Hotels to visit: 55 106 151 197 241 280 330 378 425 476 531 598 671 728 775 827 906 
Total penalty: 3534
